moveit_ros/hybrid_planning/CMakeLists.txt

```
@@ -68,25 +68,35 @@ rclcpp_components_register_nodes(moveit_hybrid_planning_manager "moveit_hybrid_p
 rclcpp_components_register_nodes(moveit_global_planner_component "moveit_hybrid_planning::GlobalPlannerComponent")
 rclcpp_components_register_nodes(moveit_local_planner_component "moveit_hybrid_planning::LocalPlannerComponent")
 
-install(TARGETS ${LIBRARIES} hybrid_planning_test_node
+# Install libraries
+install(TARGETS ${LIBRARIES}
+  EXPORT export_${PROJECT_NAME}
   ARCHIVE DESTINATION lib
   LIBRARY DESTINATION lib
-  RUNTIME DESTINATION lib/${PROJECT_NAME}
+  RUNTIME DESTINATION bin
   INCLUDES DESTINATION include)
 
+# Install binary
+install(TARGETS hybrid_planning_test_node
+  ARCHIVE DESTINATION lib
+  LIBRARY DESTINATION lib
+  RUNTIME DESTINATION bin
+  DESTINATION lib/${PROJECT_NAME}
+)
+
 install(DIRECTORY ${THIS_PACKAGE_INCLUDE_DIRS} DESTINATION include)
 
 install(DIRECTORY test/launch DESTINATION share/${PROJECT_NAME})
 install(DIRECTORY test/config DESTINATION share/${PROJECT_NAME})
 
+ament_export_include_directories(include)
+ament_export_libraries(export_${PROJECT_NAME})
```

```suggestion
ament_export_libraries(${PROJECT_NAME}Targets)
```

@JafarAbdi pointed out to me that this naming convention is more standard cmake than the export_ prefix we have been using.